QuizAPI/Controllers/QuizController.cs

```diff
@@ -1,19 +1,137 @@
-﻿using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Mvc;
+﻿using Microsoft.AspNetCore.Mvc;
 using QuizAPI.Model;
+using QuizAPI.Utils;
+using QuizAPI.DTOs;
 
 namespace QuizAPI.Controllers
 {
-    [Route("api/[controller]")]
-    [ApiController]
-    public class ValuesController : ControllerBase
-    {
-        TriviapiDBContext _context = new TriviapiDBContext();
-
-        [HttpGet(Name = "test")]
-        public IActionResult testMe()
-        {
-            return new ObjectResult(_context.Questions.Find(1));
-        }
-    }
+	[Route("api/[controller]")]
+	[ApiController]
+	public class QuizController : ControllerBase
+	{
+		TriviapiDBContext _context = new TriviapiDBContext();
+		ValueToIdUtil _valueToIdUtil = new ValueToIdUtil();
+
+		[HttpGet("{id}")]
+		public IActionResult testMe(int id)
+		{
+			Question? question = _context.Questions.Find(id);
+
+			if (question == null)
+			{
+				return NotFound();
+			}
+
+			return Ok(_context.Questions.Find(id));
+		}
+
+		[HttpPatch("{id}")]
+		public IActionResult updateQuestion(int id, [FromBody] QuestionDTO questionPatches)
+		{
+			Question? questionToChange = _context.Questions.Find(id);
```

Comment:
This code is repeated in  the patch and put. Can we maybe make a QuestionExists function?